CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Changelog
 
-## v1.5.2
+## v1.5.3 - 2021-08-13
+
+### Dependencies
+
+- Dependencies: pin `sqlalchemy` to minor version 1.3 since `sqlalchemy==1.4` breaks our compatibility
+- Dependencies: put upper limit on `psycopg2-binary` since `psycopg2-binary==2.9` breaks out test environment
+- Dependencies: fix `pymatgen` imports
```

Comment:
Maybe you can take out the multiple "dependencies" from each change? Haha it seems a bit repetitive, specially since you already have the header.